Sources/Html/Attributes.swift

```diff
@@ -572,6 +572,27 @@ public enum InputType: String {
   case week
 }
 
+public struct Accept: RawRepresentable {
+  public let rawValue: String
+
+  public init(rawValue: String) {
+    self.rawValue = rawValue
+  }
+
+  public static let audio = Accept(rawValue: "audio/*")
+  public static let image = Accept(rawValue: "image/*")
+  public static let video = Accept(rawValue: "video/*")
+
+  public static func file(_ type: String) -> Accept {
+    return .init(rawValue: type.hasPrefix(".") ? type : "." + type)
+  }
+
+  public static func mime(_ type: MediaType) -> Accept {
+    let mimeTypeOnly = type.description.split(separator: ";")[0]
```

Comment:
This bit is necessary to drop the `; charset=` bit, which unfortunately breaks the `accept` attribute (at least in Chrome and Safari).